tests/e2e/e2e_test.go

```diff
@@ -354,6 +354,46 @@ var _ = Describe("odoe2e", func() {
 
 				Expect(getMntPath).To(ContainSubstring("/mnt/pv2"))
 			})
+
+			It("should be able to unmount the storage using the storage name", func() {
+				runCmd("odo storage unmount pv2")
+
+				// Verify with deploymentconfig
+				getDc := runCmd("oc get dc/php-" + appTestName + " -o go-template='" +
+					"{{range .spec.template.spec.containers}}" +
+					"{{range .volumeMounts}}{{.name}}{{end}}{{end}}'")
+
+				Expect(getDc).NotTo(ContainSubstring("pvc2"))
```

Comment:
shouldn't it be `pv2`?

Reply:
Oops that's why the test failed, fixing it